### DISCUSSION TITLE
Welcome to AIHub Discussions!

### DISCUSSION THREAD

**Discussion by classfang** [classfang/AIHub#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tanjingdong** [comment]
看起来不错。不知道为什么star这么少。按理说并没有找到更好的同类替代。点个star支持一下吧。

**Reply by classfang** [reply]
没做推广，可能自然流量比较少吧 😊

**Comment by scsioShuang** [comment]
怎么导入模型
